src/multiparser.js

```diff
@@ -294,16 +294,22 @@ function isStyledJsx(path) {
  * styled.button`color: red`
  * or
  * Foo.extend`color: red`
+ * or
```

Comment:
could you remove all the `or`, they are not very useful now that we have 4 different ones :p